### PR TITLE
keep dialect xsd up to date

### PR DESCRIPTION
**Pull request by jsun28**
Update dev-2021.4's dialect xsd as latest near, changes are from near file history 6-10(5 revisions in total)

### CODE REVIEW

File: validation/tdd_latest.xsd [hunk 1]
```diff
@@ -209,12 +209,16 @@ as passed-in parameters.
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="BalancedIIF"/>
+      <xs:enumeration value="BalancedIIF_"/>
+      <xs:enumeration value="BalancedIF"/>
```
Comment (rosswbrown) [inline validation/tdd_latest.xsd:213]:
```suggestion
      <xs:enumeration value="BalancedIIF_"/> <!-- added 2021.4 -->
      <xs:enumeration value="BalancedIF"/> <!-- added 2021.4 -->
```
I put this comment on the other branch as well.  If for some reason comments in these XSDs aren't valid then ignore this feedback.  The packager tests should tell us either way.

Comment (jsun28) [reply validation/tdd_latest.xsd]: tested those comments with the packager, it works.

File: validation/tdd_latest.xsd [hunk 2]
```diff
@@ -209,12 +209,16 @@ as passed-in parameters.
...
       <xs:enumeration value="BalancedIIF"/>
+      <xs:enumeration value="BalancedIIF_"/>  <!-- added 2021.4 -->
+      <xs:enumeration value="BalancedIF"/>  <!-- added 2021.4 -->
     </xs:restriction>
   </xs:simpleType>
   <xs:simpleType name="IfThenElseApproach-ST">
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="IIF"/>
+      <xs:enumeration value="IIF_"/>
```
Comment (rosswbrown) [inline validation/tdd_latest.xsd:220]: Can you add the comments from 2021.3 as well here.  As long as you keep those lines identical it'll make the merge to/from master a bit easier.

Comment (jsun28) [reply validation/tdd_latest.xsd]: Done, thanks!

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**